djangocms_history/serializer.py

```diff
@@ -0,0 +1,27 @@
+from django.core.serializers.python import Serializer as PythonSerializer
+from django.utils.encoding import is_protected_type
+import json
+
+class PythonSerializerWithDetectNestedJsonField(PythonSerializer):
+
+    """
```

Comment:
Please fix docstring typo - we don't need extra blank lines here - check other classes in history addon